devtools/aesh/src/main/java/io/quarkus/cli/commands/InstallExtensionCommand.java

```diff
@@ -0,0 +1,78 @@
+package io.quarkus.cli.commands;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.aesh.command.Command;
+import org.aesh.command.CommandDefinition;
+import org.aesh.command.CommandException;
+import org.aesh.command.CommandResult;
+import org.aesh.command.invocation.CommandInvocation;
+import org.aesh.command.option.Arguments;
+import org.aesh.command.option.Option;
+import org.aesh.io.Resource;
+import org.aesh.selector.MultiSelect;
+
+import io.quarkus.dependencies.Extension;
+import io.quarkus.devtools.commands.AddExtensions;
+import io.quarkus.devtools.commands.data.QuarkusCommandException;
+import io.quarkus.devtools.commands.data.QuarkusCommandOutcome;
+import io.quarkus.devtools.project.QuarkusProject;
+import io.quarkus.platform.tools.config.QuarkusPlatformConfig;
+
+@CommandDefinition(name = "install", description = "Install extensions to a project")
+public class InstallExtensionCommand implements Command<CommandInvocation> {
```

Comment:
We won’t actually be doing install, just adding extension. 

add, maybe ?

Reply:
Yeah, as I mentioned earlier as well I used some names that were brought up in a thread I read. I thought install worked well, but we can use add too.